homeassistant/components/media_player/__init__.py

```diff
@@ -512,6 +528,22 @@ def shuffle(self):
         """Boolean if shuffle is enabled."""
         return None
 
+    @property
+    def group_candidates(self):
```

Comment:
I suggest we leave this out for now. It can be considered to be added later.

Reply:
Sure, I can just leave that out :)

Reply:
OK, I just dropped the `group_candidates` attribute for now. Let's wait if people actually need it.